lib/utils/general-utils.js

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
don't think this belong here, this is used only by ControllerWrapper, so it should be there